tests/filecheck/dialects/emitc/emitc_types.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: XDSL_ROUNDTRIP
+
+// Test the emitc array type
+// CHECK: func.func public @identity_array(%input : !emitc.array<2x2xf32>) -> !emitc.array<2x2xf32> {
+// CHECK-NEXT:   func.return %input : !emitc.array<2x2xf32>
+// CHECK-NEXT: }
+func.func public @identity_array(
+  %input: !emitc.array<2x2xf32>
+) -> !emitc.array<2x2xf32> {
+  
+  func.return %input : !emitc.array<2x2xf32>
+}
```

Comment:
Why not just a "test.op" with an attribute in a dictionary?

Reply:
Also, can you please add an MLIR roundtrip test, also with a test op

Reply:
sorry messed up my update this morning, should be there now